models/reports.yaml

```diff
@@ -8,85 +8,103 @@ models:
-                .table-container {
-                    margin-bottom: 3em;
-                }
+            .table-container {
+                margin-bottom: 3em;
+            }
 
-                .table-container table {
-                    border-collapse: collapse;
-                    width: 50%;
-                }
+            .table-container table {
+                border-collapse: collapse;
+                width: 50%;
+            }
 
-                .table-container th,
-                .table-container td {
-                    border: 1px solid #dddddd;
-                    text-align: left;
-                    padding: 0.5em;
-                }
+            .table-container th,
+            .table-container td {
+                border: 1px solid #dddddd;
+                text-align: left;
+                padding: 0.5em;
+            }
 
-                .table-container th {
-                    background-color: #f2f2f2;
-                }
+            .table-container th {
+                background-color: #f2f2f2;
+            }
 
-                img {
-                    max-width: 100%;
-                    height: auto;
-                    display: block;
-                    margin: 1em auto;
-                }
+            img {
+                max-width: 100%;
+                height: auto;
+                display: block;
+                margin: 1em auto;
+            }
             </style>
         </head>
 
-        <body style="text-align:center">
-            <h1 style="color: green">
-                Testing
-            </h1>
+        <body style="text-align: center">
+            <h1 style="color: green">Testing</h1>
+            {% set query_models = ["models/clusters", "models/overall_stats", "models/clusters_sorted_by_size", "models/clusters_sorted_by_dist", "models/top_max_degree_nodes", "models/edges_graph", "models/edges_biggest_cluster", "models/id_type_count", "models/id_count"] %}
+            {% for qmodel in query_models %}{% with _ = this.DeRef(qmodel) %}{% endwith %}{% endfor %}
             {% verbatim %}
             <h2>ID Graph</h2>
             <p>ID Graph contains top clusters sorted by their size. Each cluster contains top nodes sorted by their degree (Only edges connecting these nodes are shown. These nodes might have more edges which are not shown here)</p>
             <img src="{{this.DbObjectName("id_graph.png")}}" alt="ID Graph">
-
             <h2>Biggest Cluster</h2>
             <p>Biggest Cluster graph contains top nodes sorted by their degree (Only edges connecting these nodes are shown. These nodes might have more edges which are not shown here)</p>
             <img src="{{this.DbObjectName("biggest_cluster.png")}}" alt="Biggest Cluster">
-
-
+            <h2>Biggest Cluster</h2>
+            <p>
+            Biggest Cluster graph contains top nodes sorted by their degree (Only
+            edges connecting these nodes are shown. These nodes might have more edges
```

Comment:
1) verbatim can be revmoed.
2) Line 46 is not required as we are already de-refing in 103.
3) {{ this.DeRef(qmodel).Name()+".json" }}
     should be replaced by
     {{this.DeRef(qmodel)}}

Stringer should erturn final filenames of JSON files, in case of report model.

Reply:
addressed

Reply:
addressed.